recce/adapter/dbt_adapter/__init__.py

```diff
@@ -684,16 +684,24 @@ def get_lineage_cached(self, base: Optional[bool] = False, cache_key=0):
                 'resource_type': source['resource_type'],
                 'package_name': source['package_name'],
                 'config': source['config'],
+                'columns': {
+                    col.get('name'): {
+                        'name': col.get('name'),
+                        'type': col.get('data_type')
+                    }
+                    for col in source['columns'].values()
```

Comment:
`source['columns']` could be None. Could we give it the default empty dict?

Reply:
Yes, I should consider it possible to be missing.